### DISCUSSION TITLE
第3节课笔记(17班)

### DISCUSSION THREAD

**Discussion by Jianfeng777** [InternLM/Tutorial#183] [笔记]
请上传第三节课的笔记在此处！

**Comment by fine-lab** [comment]
第三节课笔记
https://blog.csdn.net/SmallZCan_t/article/details/135466695?spm=1001.2014.3001.5501

**Comment by Jianfeng777** [comment]
https://blog.csdn.net/jianfengkexin/article/details/135451548?csdn_share_tail=%7B%22type%22%3A%22blog%22%2C%22rType%22%3A%22article%22%2C%22rId%22%3A%22135451548%22%2C%22source%22%3A%22jianfengkexin%22%7D

**Comment by Alisa742** [comment]
https://blog.csdn.net/weixin_53222965/article/details/135534751

**Comment by zcg-joker** [comment]
[第三节课笔记](https://blog.csdn.net/m0_55764641/article/details/135481142)

**Comment by l241025097** [comment]
第三节课笔记
![03基于InternLM和LangChain搭建你的知识库读书笔记](https://github.com/InternLM/tutorial/assets/31737569/381c21e5-9cd9-44fd-9884-24ed88c99a11)

**Comment by AxrLH** [comment]
第三次课笔记

https://blog.csdn.net/Axr_Samuel/article/details/135541467?csdn_share_tail=%7B%22type%22%3A%22blog%22%2C%22rType%22%3A%22article%22%2C%22rId%22%3A%22135541467%22%2C%22source%22%3A%22Axr_Samuel%22%7D

**Comment by futtttttttttttt** [comment]
https://zhuanlan.zhihu.com/p/677742342

**Comment by JACK626** [comment]
https://www.yuque.com/xiaotuanyue/mdr8d5/fav400bga7e5r811?singleDoc#
